lib/galaxy/job_execution/actions/post.py

```diff
@@ -53,12 +53,11 @@ def execute(cls, app, sa_session, action, job, replacement_dict, final_job_state
         try:
             history_id_encoded = app.security.encode_id(job.history_id)
             link_invocation = None
+            galaxy_url = app.config.galaxy_external_url or app.config.galaxy_infrastructure_url
             if job.workflow_invocation_step:
                 invocation_id_encoded = app.security.encode_id(job.workflow_invocation_step.workflow_invocation_id)
-                link_invocation = (
-                    f"{app.config.galaxy_infrastructure_url}/workflows/invocations/report?id={invocation_id_encoded}"
```

Comment:
Hmm... well that was problematic 😨. I agree we should have this option but we should just disable this link if the config option is not set IMO. Things should just degrade well if the new option is not set and I think that means either serializing the request url with the job or dropping it all together.